src/components/Match/CardList.tsx

```diff
@@ -18,88 +15,316 @@ interface Block {
               backgroundColor: "primary.main",
               boxShadow: 5,
               marginTop: theme.spacing(2),
-              textAlign: 'center',
+              textAlign: "center",
+              transition: "transform 0.3s ease-in-out",
+              "&.delete-animation": {
+                transform: "scale(0)",
+              },
               "@media screen and (max-width: 426px)": {
-                width: theme.spacing(40),
+                width: "auto",
               },
             }}
           >
-            <Typography
-              sx={{
-                marginTop: theme.spacing(3),
-                fontSize: theme.spacing(4),
-                fontFamily: "Platypi",
-                fontWeight: 600,
-              }}
-            >
-              Match Info:
-            </Typography>
-            <Typography
-              sx={{
-                fontFamily: "Platypi",
-                fontSize: theme.spacing(3),
-                marginTop: theme.spacing(2),
-              }}
-            >
-              {block.firstTeam} - {block.secondTeam}
-            </Typography>
-            <Typography
-              sx={{
-                fontFamily: "Platypi",
-                fontSize: theme.spacing(3),
-                marginTop: theme.spacing(2),
-              }}
-            >
-              Number of tickets - "{block.tickets}"
-            </Typography>
-            <Typography
-              sx={{
-                fontFamily: "Platypi",
-                fontSize: theme.spacing(3),
-                marginTop: theme.spacing(2),
-              }}
-            >
-              Field - "{block.stadium}"
-            </Typography>
-            <Button
-              onClick={() => removeBlock(block.id)}
-              sx={{
-                marginLeft: "auto",
-                marginRight: "auto",
-                marginTop: theme.spacing(3),
-                marginBottom: theme.spacing(3)
-              }}
-            >
-              Delete
-            </Button>
+            {editMode === block.id ? (
+              <>
+                <Typography
+                  sx={{
+                    marginTop: theme.spacing(3),
+                    fontSize: theme.spacing(4),
+                    fontFamily: "Platypi",
+                    fontWeight: 600,
+                  }}
+                >
+                  Editing Mode:
+                </Typography>
+                <TextField
+                  value={editedBlock.firstTeam || ""}
+                  variant="standard"
+                  onChange={(e) => handleChange("firstTeam", e.target.value)}
+                  sx={{
+                    width: "70%",
+                    fontFamily: "Platypi",
+                    fontSize: theme.spacing(3),
+                    marginTop: theme.spacing(2),
+                    marginLeft: theme.spacing(5),
+                    marginRight: 0,
+                  }}
+                />
+                <TextField
```

Comment:
Don`t you want to move textfield to a separate file and pass different props as you use it many times?